### PR TITLE
Move status table to cron package

### PR DESCRIPTION
**Pull request by lunny**
As title.

### CODE REVIEW

File: modules/cron/cron.go [hunk 1]
```diff
@@ -12,80 +12,107 @@ import (
 	"code.gitea.io/gitea/models"
 	"code.gitea.io/gitea/modules/log"
 	"code.gitea.io/gitea/modules/setting"
+	"code.gitea.io/gitea/modules/sync"
```
Comment (techknowlogick) [inline modules/cron/cron.go:14]: * copyright header
* order of imports
* path to cron package may need to be updated (it is now github.com/gogs/cron)

Comment (lunny) [reply modules/cron/cron.go:14]: This is only a refactor of status table. I don't understand what did you mean.

Comment (techknowlogick) [reply modules/cron/cron.go:14]: Sorry, GitHub doesn't let me comment on lines that aren't edited. I see your point, and agree that copyright header probably doesn't need to be added.

The other items I listed are github library import comes before code.gitea.io imports, and also the github import needs to be updated to use correct path (gogits org renamed to gogs)

Comment (lunny) [reply modules/cron/cron.go:14]: done.

**Comment by lunny** [comment]
@techknowlogick done.

**Comment by lunny** [comment]
based on #7394

**Review by techknowlogick — APPROVED** [review]

**Review by richmahn — APPROVED** [review]
LGTM

**Pull request merged**